focolab/minimo
# Issue 13

### Purpose ###
Closes #13. (Also raised by reviewer at https://github.com/openjournals/joss-reviews/issues/2979.)

### Testing ###
1. Build and launch locally with `docker-compose up -d --build --force-recreate`
2. Go to `https://minimo.localhost/submit-data-boxuploader`
3. Verify that `Number of experiments` is set to `1`
4. Verify that one experiment metadata form is displayed below dropdown
5. Verify that `0` is not present in `Number of experiments` dropdown
6. Change `Number of experiments` value to `2`
7. Verify that two experiment metadata forms are displayed below dropdown
8. Change `Number of experiments` value back to `1`
9. Verify that one experiment metadata form is displayed below dropdown
10. Complete metadata form, select data for upload, and hit `submit`
11. Go to metadata browser and confirm that metadata row uploaded in previous step is visible
